zbus_macros/src/utils.rs

```diff
@@ -55,7 +55,7 @@ pub fn is_blank(s: &str) -> bool {
 /// Standard annotation `org.freedesktop.DBus.Property.EmitsChangedSignal`.
 ///
 /// See <https://dbus.freedesktop.org/doc/dbus-specification.html#introspection-format>.
-#[derive(Debug, Default)]
+#[derive(Debug, Default, Clone)]
```

Comment:
I wouldn't have pointed this one out but since you're going to be fixing the commits, I think best to split this into a separate one too that comes before this ("✨ Adds iface support for EmitsChangedSignal flag.") commit.